tox.ini

```diff
@@ -15,8 +15,23 @@ setenv =
 commands =
      py.test -v --timeout=30 --cov=opsdroid --cov-report term-missing:skip-covered --cov-report=xml:cov.xml {posargs}
 deps =
-     -r{toxinidir}/requirements.txt
-     -r{toxinidir}/requirements_test.txt
+	flake8==3.8.1
```

Comment:
I think these should be an extra named `test` or similar? (Which isn't installed by `all`).

Reply:
I thought since all tests are gonna be using tox anyway this would reduce some clutter in setup.py

Reply:
You don't have to run the tests in tox, so it's good to provide an easy way for people to install the requirements outside of tox.